src/toolbar.scss

```diff
@@ -21,7 +20,7 @@ $title-toolbar-height: 2.75rem;
 }
 
 @mixin toolbarAuto() {
```

Comment:
Do we need all these mixins as they're used only once?

Reply:
As toolbar is used in other components like table, facets and dynamicPage, there may be chance to reuse.